### DISCUSSION TITLE
Welcome to docker-jmkvpropedit Discussions!

### DISCUSSION THREAD

**Discussion by jlesage** [jlesage/docker-jmkvpropedit#1] [Announcements]
## 👋 Welcome!
Discussions has been enabled for this project.  This is a place to connect with other members of the community.

Here, you can:
  * Ask questions about:
    * How to setup the container.
    * How to use the application.
    * Problems you might with your specific setup.
    * etc.
  * Share ideas.
  * Engage with other community members.

Welcome others and be open-minded. Remember, this is a community we build together 💪.

**Comment by neuroryuu** [comment]
Hello, thanks for your work, you know what happens to me:
When selecting several subtitles and audios a1,a2,a3 s1,s2,s3,s4 and the file only has s1,s2,s3 it will give this error:
If it detects a file with more audio or subtitles, it gives an error
The problem is that it becomes a manual task, it may skip if it does not find a subtitle or audio.


Currently I have to make a .bat in Windows to eliminate audios and subtitles but having so many files the system crashes or stops

**Reply by jlesage** [reply]
Discussed here: https://github.com/jlesage/docker-jmkvpropedit/issues/2
